packages/fxa-auth-client/lib/hawk.ts

```diff
@@ -46,7 +46,7 @@ export async function deriveHawkCredentials(token: hexstring, context: string) {
 
 function parseUri(input: string) {
   const parts = input.match(
-    /^([^:]+)\:\/\/(?:[^@/]*@)?([^\/:]+)(?:\:(\d+))?([^#]*)(?:#.*)?$/
+    /^([^:]+):\/\/(?:[^@/]*@)?([^/:]+)(?::(\d+))?([^#]*)(?:#.*)?$/
```

Comment:
errr is this a linting fix?

Reply:
Yep this was useless escape.

Reply:
@chenba Do you have any concerns with this?

Reply:
I do not.  👍